runtime/rococo/src/lib.rs

```diff
@@ -857,6 +867,111 @@ impl pallet_bridge_grandpa::Config<WococoGrandpaInstance> for Runtime {
 	type WeightInfo = pallet_bridge_grandpa::weights::RialtoWeight<Runtime>;
 }
 
+// Instance that is 'deployed' at Wococo chain. Responsible for dispatching Rococo -> Wococo messages.
+pub type AtWococoFromRococoMessagesDispatch = pallet_bridge_dispatch::DefaultInstance;
+impl pallet_bridge_dispatch::Config for Runtime {
```

Comment:
```suggestion
pub type AtWococoFromRococoMessagesDispatch = pallet_bridge_dispatch::DefaultInstance;
impl pallet_bridge_dispatch::Config<AtWococoFromRococoMessageDispatch> for Runtime {
```

I think it's better to be explicit here, helps in the future in case we move things around.